src/internal/connector/sharepoint/site_page_test.go

```diff
@@ -0,0 +1,71 @@
+package sharepoint
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/alcionai/corso/src/internal/tester"
+	"github.com/alcionai/corso/src/pkg/account"
+)
+
+type SharePointPageSuite struct {
+	suite.Suite
+
+	creds account.M365Config
+}
+
+func (suite *SharePointPageSuite) SetupSuite() {
+	t := suite.T()
+	a := tester.NewM365Account(t)
```

Comment:
Setup should do a pre-check for the presence of the m365 env vars.  Or TestSharePointPageSuite could do it.
```suggestion
	t := suite.T()
	tester.MustGetEnvSets(t, tester.M365AcctCredEnvs)
	
	a := tester.NewM365Account(t)
```

Reply:
Resolved within [fab30b4](https://github.com/alcionai/corso/pull/2178/commits/fab30b4775cdfb82f3d998a27cea9e438b08dfb2)